RealEstateAgencyApp.AppHost/Program.cs

```diff
@@ -0,0 +1,11 @@
+﻿var builder = DistributedApplication.CreateBuilder(args);
+
+var mysql = builder.AddMySql("mysql");
+
+var mysqlDb = mysql.AddDatabase("mysqldb");
+
+var api = builder.AddProject<Projects.RealEstateAgencyApp_API>("realestate-api")
+                 .WithReference(mysqlDb, "mysqldb")
+                 .WaitFor(mysqlDb);
```

Comment:
Можно не считывать в переменную api, так как она не используется.
```suggestion
builder.AddProject<Projects.RealEstateAgencyApp_API>("realestate-api")
    .WithReference(mysqlDb, "mysqldb")
    .WaitFor(mysqlDb);
```